.github/CONTRIBUTING.md

```diff
@@ -39,7 +39,7 @@ you can run tests by making use of the `Makefile` and
 From the project root, call:
 
 - `make test` to run tests and coverage
-- `make pep` to run style checks
+- `pre-commit run` to run style checks (Ruff and some additional hooks)
```

Comment:
Shouldn't this better be `pre-commit run -a`?

Reply:
Yes, otherwise it will only run against staged files.